packages/ndla-licenses/src/getCopyString.ts

```diff
@@ -162,15 +170,21 @@ export const figureApa7CopyString = (
   license: string | undefined,
   ndlaFrontendDomain: string | undefined,
   t: TranslationFunction,
+  locale: string,
 ): string => {
   const titleString = getValueOrFallback(title, t('license.copyText.noTitle')) + ', ';
   const yearString = date ? getYearString(date) : '';
-  const creators = getCreditString(copyright, { byPrefix: true }, t);
+  const creators = getCreditString(copyright, { byPrefix: true, combineCreatorsAndRightsholders: true }, t);
   const url = `(${path ? ndlaFrontendDomain + path : src}). `;
-  const licenseString = license ? license + '.' : '';
 
-  // Ex: Tittel, 1914, av Nordmann, O. NDLA. (https://ndla.no/urn:resource:123). CC-BY-SA-4.0.
-  return titleString + yearString + creators + 'NDLA. ' + url + licenseString;
+  const licenseAbbreviation = license && getLicenseByAbbreviation(license, locale).abbreviation;
+  const isCreativeCommonsLicense = licenseAbbreviation && licenseAbbreviation.slice(0, 3) === 'CC ';
```

Comment:
Ane ikkje kva som er dyrest av slice og startsWith, men slice lager en kopi som uansett berre kastes her.
```suggestion
  const isCreativeCommonsLicense = licenseAbbreviation?.startsWith('CC ');
```

Reply:
Det virket som en bedre løsning 😄